pages/asru/views/index.jsx

```diff
@@ -0,0 +1,32 @@
+import React, { Fragment} from 'react';
+import { connect } from 'react-redux';
+import { Snippet, FormLayout, Header } from '@asl/components';
+import { Button } from '@ukhomeoffice/react-components';
+
+const Page = ({ asru, baseUrl }) => {
+  return (
+
+    <Fragment>
+      {
+        asru.map(profile => (
+          <p key={`${profile.id}`}>{profile.firstName} {profile.lastName}
+            <form method='POST' action={`${baseUrl}/delete`}>
```

Comment:
pretty sure you can't have a `form` as a child of a `p`

Reply:
in Fragment ?

Reply:
You don't need to wrap it in anything if its single element - just return the `<form>`. You _might_ need to add `display: block`

Reply:
The `<p>` is still necessary for structural reasons.

You need to reorder the tags so that the `<form>` wraps the `<p>`. Otherwise React will throw an error when run in production mode.

Reply:
done